src/NodeBuilder.js

```diff
@@ -255,7 +256,7 @@ class NodeJsBuilder {
   // 4. process mainAppFile (gzip, base64 encode it) - could be a placeholder file
   // 5. kick off ./configure & build
   buildFromSource(uploadBuild, cache, container, arch, ptrCompression) {
-    const makeArgs = isWindows ? ['x64', 'noetw', 'no-cctest'] : [`-j${os.cpus().length}`];
+    const makeArgs = isWindows ? ['x64', 'no-cctest'] : [`-j${os.cpus().length}`];
```

Comment:
Support for `etw` (Event Tracing for Windows) has been removed since nodejs v19.0.0 onward. https://github.com/nodejs/node/pull/43652
It's mentioned in the nodejs [v19 changelog](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V19.md) as well.